### DISCUSSION TITLE
Use an alternative to Heroku to deploy

### DISCUSSION THREAD

**Discussion by ang3lo-azevedo** [tjarbo/discord-moodle-bot#252] [Ideas]
Now that Heroku is paid is there any possibility to add a free alternative to deploy the project like Vercel?

**Comment by tjarbo** [comment]
It is in general a good idea to add new hosting platforms for more flexibility. But as written in the documentation (I hope so 👀 ... will need to check), the notification service requires by design an always-on dyno. Therefore, even before they stopped providing a free dyno, running the notification service on Heroku was not free. But I added Heroku, as it is part of the [GitHub Student Developer Pack](https://education.github.com/pack) with enough credits to run a "Basic" dyno and because Moodle is mainly used by universities and schools, the common user can easily get access to the pack.
